src/pages/ReimbursementAccount/BeneficialOwnerInfo/substeps/BeneficialOwnerCheckUBO.tsx

```diff
@@ -43,27 +43,22 @@ function BeneficialOwnerCheckUBO({title, onSelectedValue, defaultValue}: Benefic
     );
 
     return (
-        <ScreenWrapper
-            testID={BeneficialOwnerCheckUBO.displayName}
```

Comment:
I think we should keep `testID` prop if it has been defined. Not sure if it will work the same with FormProvider